README.md

```diff
@@ -113,6 +113,23 @@ The Grafana dashboards page should open for you automatically. If not, navigate
 to `localhost:13300/dashboards` in your browser and begin exploring the
 available dashboards.
 
+### Visualising cbbackupmgr stats files
+Ensure that you have access to either a backup archive or a cbbackupmgr collect zip
+file.
+
+Start Promtimer:
+
+```bash
+promtimer --stats-archive-path <path-to-backup-archive-or-zip>
+```
+
+The `path-to-backup-archive-or-zip` specifies the path to either a backup archive or to
+a cbbackupmgr collect zip file.
+
+The Grafana dashboards page should open for you automatically. If not, navigate
+to `localhost:13300/dashboards` in your browser and begin exploring the
+available dashboards.
+
```

Comment:
Should we document that this only works if you have access to the private `backup` repository.

Reply:
Forgot this, my bad